package.json

```diff
@@ -61,7 +61,7 @@
   "dependencies": {
     "@babel/runtime": "^7.10.3",
     "genesys-cloud-service-discovery-web": "3.0.242",
-    "query-string": "^5.0.0"
+    "query-string": "7.1.3"
```

Comment:
Can you use the cli to remove `@types/query-string` from package and lockfile?  Looks like query-string now publishes it's own types so we shouldn't need to add the external package.

https://www.npmjs.com/package/@types/query-string